js/app.js

```diff
@@ -70,12 +63,122 @@ function getDate() {
 }
 
 function Controls() {
-    var LocationsDropdown = {
+    let showModal = false;
+    let canteens = [];
+
+    function openStreetMap() {
+        const greenIcon = new L.Icon({
+            iconUrl: 'https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-2x-green.png',
+            shadowUrl: 'https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/images/marker-shadow.png',
```

Comment:
What is this `shadowUrl` for?

Reply:
I think, it was. concercing the shadow of the marker, which makes it look 3D.
However, i used CSS now instead of this icon.

Reply:
Aha, ok :)